test/services/supergroup_sections_test.rb

```diff
@@ -6,118 +6,51 @@
-      stub_tagged_content_search(policy_supergroup, 'policy_paper')
-
-      expected = [
-        {
-          link: {
-            text: 'Tagged Content Title',
-            path: '/government/tagged/content'
-          },
-          metadata: {
-            public_updated_at: '2018-02-28T08:01:00.000+00:00',
-            organisations: 'Tagged Content Organisation',
-            document_type: 'Policy paper'
-          }
-        }
-      ]
-
-      assert_equal expected, sections.section_document_list(policy_supergroup)
+  let(:supergroup_sections) { SupergroupSections::Sections.new(taxon_id, base_path) }
+
+  describe '#sections' do
+    before(:all) do
+      PolicyAndEngagement.any_instance.stubs(:tagged_content).with(taxon_id).returns(section_tagged_content_list('case_study'))
+      Service.any_instance.stubs(:tagged_content).with(taxon_id).returns(section_tagged_content_list('form'))
+      GuidanceAndRegulations.any_instance.stubs(:tagged_content).with(taxon_id).returns(section_tagged_content_list('guide'))
+      NewsAndCommunications.any_instance.stubs(:tagged_content).with(taxon_id).returns([])
+      Transparency.any_instance.stubs(:tagged_content).with(taxon_id).returns([])
+      @sections = supergroup_sections.sections
     end
 
-    it 'returns document list for services supergroup' do
-      stub_tagged_content_search(services_supergroup, 'form')
-
-      expected = [
-        {
-          link: {
-            text: 'Tagged Content Title',
-            path: '/government/tagged/content',
-            description: 'Description of tagged content'
-          }
-        }
-      ]
-
-      assert_equal expected, sections.section_document_list(services_supergroup)
+    it 'returns a list of sections with tagged content' do
+      assert 5, @sections.length
     end
 
-    it 'returns document list for guides' do
-      stub_tagged_content_search(guidance_and_regulation_supergroup, 'guide')
-
-      expected = [
-        {
-          link: {
-            text: 'Tagged Content Title',
-            path: '/government/tagged/content',
-            description: 'Description of tagged content',
-          },
-          metadata: {
-            document_type: 'Guide'
-          }
-        }
-      ]
-
-      assert_equal expected, sections.section_document_list(guidance_and_regulation_supergroup)
+    it 'returns a list of supergroup details' do
+      @sections.each do |section|
+        assert_equal(%i(title documents see_more_link show_section), section.keys)
+      end
     end
-  end
-
-  describe '#section_finder_link' do
-    it 'returns info for link to finder page' do
-      expected = {
-        text: 'See all guidance and regulation',
-        url: '/search/advanced?group=guidance_and_regulation&topic=%2Fbase%2Fpath'
-      }
 
-      assert_equal expected, sections.section_finder_link(guidance_and_regulation_supergroup.name)
-    end
-  end
+    it 'each section has a title' do
+      exptected_titles = [
```

Comment:
Typo. Should be "expected"

Reply:
I have added a spell checker plugin to my text editor